crates/bitwarden/src/tool/generators/username_forwarders/mod.rs

```diff
@@ -0,0 +1,22 @@
+pub(super) mod addyio;
+pub(super) mod duckduckgo;
+pub(super) mod fastmail;
+pub(super) mod firefox;
+pub(super) mod forwardemail;
+pub(super) mod simplelogin;
+
+fn format_description(website: &Option<String>) -> String {
+    let description = website
+        .as_ref()
+        .map(|w| format!("Website: {w}. "))
+        .unwrap_or_default();
+    format!("{description}Generated by Bitwarden.")
+}
+
+fn format_description_ff(website: &Option<String>) -> String {
+    let description = website
+        .as_ref()
+        .map(|w| format!("{w} - "))
+        .unwrap_or_default();
+    format!("{description}Generated by Bitwarden.")
+}
```

Comment:
@kspearrin do you remember why Firefox forwarder uses a different description than the other formatters?

Reply:
I do not, sorry.

Reply:
👍, let's keep it for now but with the knowledge we can revisit this in the future.